Source/WTF/wtf/cocoa/URLCocoa.mm

```diff
@@ -52,9 +52,12 @@ - (BOOL)_web_looksLikeIPAddress;
 
 RetainPtr<CFURLRef> URL::emptyCFURL()
 {
-    // We use the toll-free bridge to create an empty value that is distinct from null that no CFURL function can create.
     // FIXME: When we originally wrote this, we thought that creating empty CF URLs was valuable; can we do without it now?
-    return bridge_cast(adoptNS([[NSURL alloc] initWithString:@""]));
+    auto result = adoptCF(CFURLCreateWithString(kCFAllocatorDefault, CFSTR(""), nullptr));
+    auto result2 = adoptNS([[NSURL alloc] initWithString:@""]);
+    if (![result2 isEqual:bridge_cast(result.get())])
```

Comment:
@aproskuryakov the fact these tests have passed just goes to show these are the same.

Reply:
I do not intend on keeping this abort code. This code was just to prove a point.

Reply:
I don't know if we had a test for this behavior, or why it was needed. Not failing any tests is a useful data point, but no proof. 

The standard of prof is fairly high here, as the benefit of changing this code at all is unclear. And even if we were certain that the change was correct, we'd want to understand that it's beneficial in some way.

Reply:
Well, it would be nearly impossible for the tests to pass if an abort() were to have happened at any time, now would it?